specification/hardwaresecuritymodules/resource-manager/readme.go.md

```diff
@@ -23,6 +23,7 @@ azure-arm: true
 batch:
   - tag: package-2021-11
   - tag: package-2018-10
+  - tag: package-2022-08-preview
```

Comment:
Can you please add the latest version contents above first in order?

Reply:
Moved latest version to the top